doc/source/rllib/package_ref/models.rst

```diff
@@ -0,0 +1,27 @@
+.. _model-reference-docs:
+
+Model APIs
+==========
+
+ModelV2 API (rllib.env.models.modelv2.ModelV2)
+++++++++++++++++++++++++++++++++++++++++++++++
+
+All RLlib neural network models have to be provided as ModelV2 sub-classes.
+
+.. autoclass:: ray.rllib.models.modelv2.ModelV2
+    :members:
+
+
+RLlib comes with two sub-classes for TF (keras) models and PyTorch models:
+
+TFModelV2 (rllib.env.models.tf.tf_modelv2.TFModelV2)
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. autoclass:: ray.rllib.models.tf.tf_modelv2.TFModelV2
+    :members:
+
+TorchModelV2 (rllib.env.models.torch.torch_modelv2.TorchModelV2)
```

Comment:
<img width="260" alt="Screen Shot 2021-12-06 at 8 44 53 PM" src="https://user-images.githubusercontent.com/4529381/144967885-36b47a5d-5809-401a-9ed3-c0107775c303.png">

the name here doesn't render properly, i would suggest removing the location string?

Reply:
Looks ok to me. I am getting weird text for the `gym.spaces.Space` typehints, like:

<img width="955" alt="image" src="https://user-images.githubusercontent.com/5839854/145413132-57dfd7a9-1fdf-4838-a6b9-6610484089c8.png">

Reply:
btw I was talking about the sidebar. 

Also we have to fix the mock spaces